app/models/recipe.rb

```diff
@@ -0,0 +1,4 @@
+class Recipe < ApplicationRecord
+  belongs_to :user
+  has_many :recipe_food
```

Comment:
- [ ] Kindly reference the earlier comment and make the same changes here as well :100: